sycl/ReleaseNotes.md

```diff
@@ -273,7 +273,7 @@ Release notes for commit range
 - Relaxed diagnostic about using virtual functions in SYCL kernels: now it is
   only emitted if a call is perfomed using virtual call mechanism, but it is not
   emitted for non-virtual calls of virtual functions. See also
-  KhronosGroup/SYCL-Docs#565. intel/llvm#114051 intel/llvm#14141
```

Comment:
Oh, I even remember that I wanted to fix that one! But apparently I had forgotten